package.json

```
@@ -6,12 +6,12 @@
   "type": "commonjs",
   "types": "types/index.d.ts",
   "scripts": {
-    "coverage": "npm run test:unit -- --coverage-report=html",
+    "coverage": "c8 --100 --reporter=html node --test --test **/*.test.js",
     "lint": "eslint",
     "lint:fix": "eslint --fix",
     "test": "npm run test:unit && npm run test:typescript",
     "test:typescript": "tsd",
-    "test:unit": "tap",
+    "test:unit": "c8 --100 node --test **/*.test.js",
```

this should be the equivalent 
```suggestion
    "test:unit": "c8 --100 node --test",
```

---

Not on node 20. Without pattern it tries to read the no-link file and fails to run the tests